app/templates/play/modal/buy-gems-modal.jade

```diff
@@ -8,7 +9,8 @@
 
     else
       img(src="/images/pages/play/modal/buy-gems-background.png")#buy-gems-background
-
+      h1(data-i18n="buy_gems.buy_gems")
```

Comment:
Could you use "play.buy_gems", instead (cause there is already a "Buy Gems" string), and then remove the one you added to the en.coffee file?